app/mailers/generic_mailer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class GenericMailer < ApplicationMailer
+  default from: %("Mailgun Sandbox" \
+    <postmaster@sandbox8f696e611a6e4906a977c007f8971322.mailgun.org>)
+
+  def drop_duty(duty)
+    @duty = duty
+    mail(to: all_users_with_name,
```

Comment:
If I remember correctly you should be able to put these text on a view of some sorts right? Nicer to separate these text.

Reply:
Oh this is subject. Hmmm I still feel somewhat meh putting this here. Okay how about a function that takes in a duty and spouts out the subject text?